helm/e2e-app-chart/templates/deployment.yaml

```diff
@@ -18,13 +18,13 @@ spec:
       affinity:
         podAntiAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
-            - labelSelector:
-                matchExpressions:
-                  - key: "app"
-                    operator: In
-                    values:
-                    - e2e-app
-              topologyKey: "kubernetes.io/hostname"
+          - labelSelector:
+              matchExpressions:
+              - key: "app"
+                operator: In
+                values:
+                - e2e-app
+            topologyKey: "kubernetes.io/ip"
```

Comment:
This is the main thing that gets fixed. See https://gigantic.slack.com/archives/C12SG2929/p1523893101000222.

Reply:
Note I tested this on the guest cluster created in CI build https://circleci.com/gh/giantswarm/aws-operator/5433.